diffsmhm/galhalo_models/sigmoid_smhm.py

```diff
@@ -125,6 +125,8 @@ def logsm_from_logmhalo_jax_kern(logm, params):
     logmstar : array-like
         Base-10 log of stellar mass.
     """
+    # smhm_k_logm is the param that fails finite diff
+    # but only when we add logsm_at_logm_crit to the other term
```

Comment:
Can you expand on this comment? right now it confuses me.

Reply:
that's an old comment that should be removed! A while ago I had what turned out to be a NaN problem caused by an issue elsewhere